lib/internal/bootstrap/node.js

```diff
@@ -114,6 +114,11 @@ if (isMainThread) {
   process.memoryUsage = wrapped.memoryUsage;
   process.kill = wrapped.kill;
   process.exit = wrapped.exit;
+
+  process._getSourceMapCache = function() {
+    const { _getSourceMapCache } = require('internal/source_map');
```

Comment:
I can't seem to find the source of this? Is it `sourceMapCacheToObject` ?

Reply:
https://github.com/nodejs/node/pull/28960/files#diff-56eb150ba9391c84a1fca2510f13d34aR21

Reply:
Sorry, I don't think I am following..I still can't find a `_getSourceMapCache` exported out of 'internal/source_map'?